### PR TITLE
add description of not allowed pattern with multiple variables

### PR DESCRIPTION
**Pull request by nadja-muller**
Describes the behavior for pattern with reused relationship variables

### CODE REVIEW

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/PatternTest.scala [hunk 1]
```diff
@@ -184,6 +184,12 @@ class PatternTest extends DocumentingTest {
           |(a)-[:REL_TYPE]->(b)
           |----
           |""")
+      p("Within a pattern it is not possible to use the same name for a relationship multiple times due to <<relationship-isomorphism, relationship isomorphism>>. The following example is not allowed:")
```
Comment (AlexicaWright) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/PatternTest.scala]:
```suggestion
      p("It is not possible to use the same for a relationship multiple times within a pattern due to <<relationship-isomorphism, relationship isomorphism>>. The following example is therefore not allowed:")
```

**Review by AlexicaWright — COMMENTED** [review]
Just a small suggestion for clarity.

**Review by AzuObs — APPROVED** [review]

**Comment by Lojjs** [comment]
[Remove deprecated 'Disallow repeated relationships in pattern expressions/pattern comprehensions'](https://trello.com/c/ppfimmOD/69-remove-deprecated-disallow-repeated-relationships-in-pattern-expressions-pattern-comprehensions)

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/PatternTest.scala [hunk 2]
```diff
@@ -169,22 +194,35 @@ class PatternTest extends DocumentingTest {
...
           |----
           |""")
+      p(
+        "It is not possible to use the same name for a relationship multiple times within a pattern due to <<relationship-isomorphism, relationship isomorphism>>. The following example is therefore not allowed:"
+      )
+      p("""[source, cypher, role=noplay]
+          |----
+          |()-[r:REL_TYPE]-()-[r:REL_TYPE]-()
+          |----
+          |""")
```
Comment (martin-neotech) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/PatternTest.scala:223]: This is the actual change...

**Pull request merged**